tools/get_release_contributors.py

```diff
@@ -1,27 +1,32 @@
 #!/usr/bin/env python3
 """
-Generates a list of Github user names who contributed to a specific release.
+Generates a list of GitHub usernames who contributed to a specific release.
 
-May need a Github Personal access token to avoid hitting rate limits.
+May need a GitHub Personal access token to avoid hitting rate limits.
 
 Usage:
 
 ## Website repo (jamulussoftware/jamuluswebsite)
 - cd jamuluswebsite
-- git log --oneline release # Find commit ID of previous version. Search for "RELEASE:"
-- /path/to/jamulussoftware/code/tools/get_release_contributors.py --from COMMIT_ID_FROM_PREVIOUS_STEP --to next-release --github-token YOUR_TOKEN --repo .
+# Find commit ID of previous version. Search for "RELEASE:"
+- git log --oneline release
+- /path/to/jamulussoftware/code/tools/get_release_contributors.py \
+    --from COMMIT_ID_FROM_PREVIOUS_STEP --to next-release \
+    --github-token YOUR_TOKEN --repo .
 
 ## Code repo (jamulussoftware/jamulus)
 - git pull -C /path/
-- ./tools/get_release_contributors.py --from r3_8_1 --to r3_8_2 --github-token YOUR_TOKEN --repo .
+- ./tools/get_release_contributors.py --from r3_8_1 --to r3_8_2 \
+    --github-token YOUR_TOKEN --repo .
 
 """
-import requests
 import argparse
 import logging
 import os
 import re
 import subprocess
+
+import requests
```

Comment:
Why is there an empty line? Also does order matter in Python? Probably not?

Reply:
No, it's a style. Linters try to order imports to avoid churn in Git (etc). Typically third party ones are separated from "native" Python ones

Reply:
Ok. Then I'd add a comment saying something like import third party libraries

Reply:
I have pretty much never seen this in Python though; it's just autoformatting that the linters suggest

Reply:
Ok. It still looks odd. Is it really common for python devs to know why an empty line is here? If yes, ok. If no, I'd prefer to at least document that somewhere (styleguide)

Reply:
Ok. Thanks for the clarification :-).